pkgs/development/tools/locate-dominating-file/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  bats,
+  bash,
+  lib,
+  stdenv,
+  getopt,
+  fetchFromGitHub,
+}: let
+  batsPkg = bats.withLibraries (p: [p.bats-support p.bats-assert]);
+  version = "v0.0.1";
```

Comment:
`version` should start with a digit - except maybe `unstable-YYYY-MM-DD` (when picking a particular VCS commit).